lib/ProMotion/table/table.rb

```diff
@@ -325,7 +323,14 @@ def table_style
       end
 
       def row_height(height, args={})
-        height = UITableViewAutomaticDimension if height == :auto
+        if height == :auto
+          if UIDevice.currentDevice.systemVersion.to_f < 8.0
+            height = args[:estimated] || 44.0
+            PM.logger.warn "Using `row_height :auto` is not supported in iOS 7 apps. Setting to #{height}."
+          else
+            height = UITableViewAutomaticDimension
+          end
+        end
         args[:estimated] ||= height unless height == UITableViewAutomaticDimension
         @row_height = { height: height, estimated: args[:estimated] || 44.0 }
       end
```

Comment:
This method got more complex, because I have to handle iOS 7's non-implementation of `UITableViewAutomaticDimension` for table row height. I'm also outputting a `warn` to keep devs happy.